ui-tests/src/test/java/org/openmrs/reference/MergeVisitsTest.java

```diff
@@ -36,13 +35,14 @@ public void setUp() {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
     public void mergeVisitsTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
```

Comment:
`Waiting the page to load` ensures that the searched patient identifier matches with the created test patient identifier. Otherwise without` waiting the page to load`, some times the comparison is made between the searched patient identifier with any identifier (`randomly picked from the db pool`) of the existing patient data present in the  `db server instance` because the `search patient indexing criteria` that searches existing patient takes a bit longer to retrieve the `created test patient data` from the `db pool`